src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/config/ResourceServerConfiguration.kt

```diff
@@ -11,13 +12,24 @@ import org.springframework.security.oauth2.server.resource.authentication.JwtGra
 
 @Configuration
 @EnableWebSecurity
-class ResourceServerConfiguration : WebSecurityConfigurerAdapter() {
-  // todo: add custom token validator which calls token verification service
+@Profile("test")
+class BasicResourceServerConfiguration : WebSecurityConfigurerAdapter() {
```

Comment:
i understand why this is here, but it makes me nervous of deploying the app with this config enabled by mistake! is there any other way to alter this config in the test code?

Reply:
I couldn't find any. I'm sure there are ways, but I'm not familiar with Spring Boot 😞

Reply:
💭 Maybe I can flip this and move to the test source set. ⌛

Reply:
Is this a blocker? I'm happy to commit to dig into this and arrive at a sensible thing separately.

Reply:
@tomsmyers Is this a blocker? I'm happy to commit to dig into this and arrive at a sensible thing separately.

Reply:
no will address seperately